patterns/components/breadcrumb.php

```diff
@@ -1,7 +1,84 @@
 <?php
-  if (function_exists('yoast_breadcrumb')) {
-  yoast_breadcrumb('
-    <nav class="breadcrumbs" role="navigation"><ul class="breadcrumbs__list">','</ul></nav>
-  ');
+function wordpress_breadcrumbs()
+{
+  $name = 'Home'; //text for the 'Home' link
+  $currentBefore = '<li class="breadcrumbs__list-item font--secondary--xs upper dib"><a class="breadcrumbs__link can-be--white">';
```

Comment:
Let's not camelCase `PHP` variable names. Please keep with the coding format used throughout the codebase.